pkg/sfu/streamallocator.go

```diff
@@ -687,10 +687,13 @@ func (s *StreamAllocator) maybeCommitEstimate() (isDecreasing bool) {
 		s.lastEstimateDecreaseTime = time.Now()
 		isDecreasing = true
 	}
+
+	if s.committedChannelCapacity > s.receivedEstimate {
+		s.params.Logger.Debugw("committing channel capacity(bps)", "from", s.committedChannelCapacity, "to", s.receivedEstimate)
+	}
```

Comment:
Logging this only on estimate decrease as there are a bunch of these log lines and not super useful. By logging before and after when there is a decrease we will get an idea of how big the drop is.